test/integration/test_integration.py

```diff
@@ -2029,7 +2024,9 @@ def forward(self, x):
         # we can re-enable this after non-functional IR is enabled in export
         # model = torch.export.export(model, example_inputs).module()
         if TORCH_VERSION_AT_LEAST_2_5:
-            model = torch.export.export_for_training(model, example_inputs).module()
+            model = torch.export.export_for_training(
+                model, example_inputs, strict=True
```

Comment:
is `strict` False by default now?

Reply:
strict=False is not by default yet, but it will be. I don't think you need to add this everywhere as in vast majority of use cases, the end graphs should be the same.

Reply:
I think @gmagogsfm was just doing a code mod change and this is just for preserving the old behaviour?